src/libs/ck-libs/NDMeshStreamer/NDMeshStreamer.ci

```diff
@@ -27,57 +29,21 @@ module NDMeshStreamer {
     entry void init(CkCallback endCb, int prio);
     entry [reductiontarget] void syncInit();
 
-    entry void receiveAtDestination(MeshStreamerMessage<dtype> *msg);
+    entry void receiveAtDestination(MeshStreamerMessageV *msg);
   };
 
   template<class dtype, class ClientType, class RouterType, int (*EntryMethod)(char *, void *) = defaultMeshStreamerDeliver<dtype,ClientType> >
-    group [migratable] GroupMeshStreamer : MeshStreamer<dtype, RouterType> {
```

Comment:
This `GroupMeshStreamer` has the support for chare groups and should be revived later, right?

Reply:
Yes.